experimenter/experimenter/theme/static_src/package.json

```diff
@@ -4,15 +4,19 @@
   "description": "",
   "private": true,
   "scripts": {
-    "build": "yarn run build:clean && yarn run build:tailwind",
+    "build": "yarn run build:clean && yarn run build:tailwind && yarn run rollup",
     "build:clean": "rm -rf ../static/css/dist",
     "build:tailwind": "env NODE_ENV=production tailwindcss --postcss -i ./src/styles.css -o ../static/css/dist/styles.css --minify",
     "dev": "env NODE_ENV=development tailwindcss --postcss -i ./src/styles.css -o ../static/css/dist/styles.css -w",
```

Comment:
We probably want `dev` to also run rollup, in case anything changes. I suggest we do someting like:

```
"scripts": {
  "dev": "npm-run-all --parallel --race dev:*",
  "dev:tailwind": "env NODE_ENV=development tailwindcss --postcss -i ./src/styles.css -o ../static/css/dist/styles.css -w",
  "dev:rollup": "rollup --watch --config rollup.config.mjs"
}
```

which requires installing `npm-run-all`